src/applications/travel-pay/tests/containers/TravelPayStatusApp.unit.spec.jsx

```diff
@@ -38,6 +39,10 @@ describe('App', () => {
     global.window.location = {
       replace: sinon.spy(),
     };
+    const mockTravelClaims = {
+      data: [{ id: 'abc' }],
```

Comment:
Can you update this to be an array of at least one "real" claim? This'll save Dan some work on the FE stuff and be more representative of what we'll get from the actual backend.